src/components/SignOut/SignOut.tsx

```diff
@@ -24,36 +23,31 @@ export const SignOut = ({ collapsed }: SignOutProps) => {
   }
 
   return (
-    <>
+    <div>
```

Comment:
@cy-len  Doesn't it add a level of complexity/ Dom depth level to use div instead of React Fragment? Why do you do that??